### PR TITLE
SB-75: Create README for backend repo

### PR DESCRIPTION
**Pull request by chunloy**
Note: This is a child ticket to SB-57-Standardize workflow for creating python repo

**What does the code do?**
Provides instructions for setting up the repo after cloning.

**What are potential issues?**
Steps for creating the database and tables will be obsolete once the database is migrated to DynamoDB.

### CODE REVIEW

**Review by dmo876 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+Note the username and password as you will need them for setting up environmet variables
+
+## Setting up .env file
+
+1. Make a copy `env.example` and rename it to `.env`
+2. Change `USERNAME` and `PASSWORD` to match the username and password from the previous step
+
+## Generating the tables
+
+In the root folder, run `flask run` to create heartbeat tables
```
Comment (yhus) [inline README.md:28]: Is this still relevant? All heartbeat logic should have moved out of this repo?

Comment (chunloy) [reply README.md]: Thanks for catching that. Yes, the heartbeat logic has been separated. Running the `flask run` command will generate the drivers and riders tables. Fixed the README.

**Review by tonyxuan2021 — APPROVED** [review]
Looks good to me. Good job!

**Review by yhus — APPROVED** [review]
🚀

**Pull request merged**